### ISSUE TITLE
A11y_Teams Toolkit Extension_Account_Account treeView_ScreenReader: NVDA not announcing extra information provided by tooltip for "Accounts".

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/vscode#148113]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;GH-TeamsToolkitExtension-Win32-Jan2022;#VS Code
;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#NVDA;

### Environment Details:
Application Name: Visual Studio Code
Extension Name: Teams Toolkit Extension
Windows version: Windows 11
Version: 1.63.2 (user setup)
NVDA version: 2021.2

### Repro Steps:

1. Launch Visual studio code
2. Install Teams toolkit
3. Tab till Teams toolkit icon and hit enter
4. Create new teams app or create using sample apps
5. Tab till "Account treeView" section and hit enter
6. Observe that NVDA not announcing extra information provided by tooltip for "Accounts".

### Actual:
NVDA not announcing extra information provided by tooltip for "Accounts".

NOTE: Same issue is reproducing in JAWS also.

### Expected:
NVDA should announce extra information provided by tooltip for "Accounts".

JAWS Attachment
![13072455_A11y_Teams Toolkit Extension_Account_Account treeView_ScreenReader](https://user-images.githubusercontent.com/85863895/165228303-22758917-39e3-400c-a57e-caf69f28568d.png)

NVDA Attachment
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/85863895/165228722-4d94aadc-f212-4c8a-aa12-328d9aeec870.png)

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. 

So Teams engineers please invest effort in filling a proper VS Code issue using our component names. Thanks!

**Comment by Ashutoshsahoo164** [comment]
Hi @isidorn Could you please provide us the right repo to log the bug?

**Comment by isidorn** [comment]
@Ashutoshsahoo164 as far as I understand you are testing the Teams toolking for the Teams extension. Please talk to the Teams team and they can help you. I work on VS Code, not on the VS Code teams extension. Thanks
